src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/SlingInitialContentBundleEntryMetaDataCollector.java

```diff
@@ -56,10 +62,13 @@ class SlingInitialContentBundleEntryMetaDataCollector {
     private final String basePath;
     private final ContentPackage2FeatureModelConverter contentPackage2FeatureModelConverter;
     private final Path newBundleFile;
-    private final Set<SlingInitialContentBundleEntryMetaData> collectedSlingInitialContentBundleEntries = new HashSet<>();
+    private final NavigableSet<SlingInitialContentBundleEntryMetaData> collectedSlingInitialContentBundleEntries = new ConcurrentSkipListSet<>();
+    private final NavigableSet<SlingInitialContentBundleEntryMetaData> defaultContentXmlEntries = new ConcurrentSkipListSet<>();
     private final AtomicLong total = new AtomicLong(0);
     private final JarFile jarFile;
 
+    private final File DEFAULT_NODE_TYPE_XML;
```

Comment:
```suggestion
    private final File EMPTY_SLING_FOLDER_XML;
```

And please rename the corresponding file as well.